package.json

```diff
@@ -1,43 +1,49 @@
 {
-	"name": "ReactNativeBase",
-	"version": "0.0.1",
-	"private": true,
-	"scripts": {
-		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"lint": "eslint src/**",
-		"test": "jest"
-	},
-	"dependencies": {
-		"humps": "^2.0.1",
-		"immutable": "^3.8.1",
-		"lodash": "^4.17.4",
-		"prop-types": "^15.5.10",
-		"react": "16.0.0-alpha.12",
-		"react-native": "0.46.2",
-		"react-native-config": "^0.5.0",
-		"react-native-navigation": "^1.1.134",
-		"react-redux": "^5.0.5",
-		"redux": "^3.7.2",
-		"redux-form": "^7.0.0",
-		"redux-immutable": "^4.0.0",
-		"redux-react-native-session": "^1.0.3",
-		"redux-thunk": "^2.2.0",
-		"validate.js": "^0.11.1"
-	},
-	"devDependencies": {
-		"babel-eslint": "^7.2.3",
-		"babel-jest": "20.0.3",
-		"babel-preset-react-native": "2.1.0",
-		"eslint": "3.19.0",
-		"eslint-config-airbnb": "15.0.1",
-		"eslint-plugin-import": "2.2.0",
-		"eslint-plugin-jsx-a11y": "5.0.3",
-		"eslint-plugin-react": "7.0.1",
-		"jest": "20.0.4",
-		"react-test-renderer": "16.0.0-alpha.12",
-		"redux-logger": "^3.0.6"
-	},
-	"jest": {
-		"preset": "react-native"
-	}
+  "name": "TargetMVD",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "lint": "eslint src/**",
```

Comment:
Adding another script with watch option could help you during development. But is up to you to decide if it's actually useful or not